### PR TITLE
ci: Fix workflow request for disabled browsers

### PR DESCRIPTION
**Pull request by joeyparrish**
You should be able to request a browser explicitly even if it is disabled by default.  This fixed the workflow logic that limited this.

### CODE REVIEW

**Comment by joeyparrish** [comment]
Running https://github.com/shaka-project/shaka-player/actions/runs/9021261305 to verify the change...

File: .github/workflows/selenium-lab-tests.yaml [hunk 1]
```diff
@@ -114,11 +114,14 @@ jobs:
...
-                (browserFilter.size == 0 ||
-                 browserFilter.has(name.toLowerCase()));
+            const thisBrowserRequested = browserFilter.has(name.toLowerCase());
+            const nothingRequested = browserFilter.size == 0;
+            const thisBrowserOnByDefault = !gridBrowserMetadata[name].disabled;
+
+            // A browser is enabled if it's requested, or if it's on by default
+            // and nothing was requested.
+            const enabled = thisBrowserRequested ||
+                (nothingRequested == 0 && thisBrowserOnByDefault);
```
Comment (alekzz) [inline .github/workflows/selenium-lab-tests.yaml:124]: nit; `nothingRequested` is boolean, maybe drop `== 0`

**Review by alekzz — APPROVED** [review]

File: .github/workflows/selenium-lab-tests.yaml (hunk 1 above)
Comment (joeyparrish) [reply .github/workflows/selenium-lab-tests.yaml:124]: Oops, good catch

Comment (joeyparrish) [reply .github/workflows/selenium-lab-tests.yaml:124]: `false == 0` is true, so the test run should still be valid.  I'll push the fix for that typo as soon as the results come in.

**Comment by joeyparrish** [comment]
The logic still isn't quite right... The workflow started all default browsers + the disabled one I explicitly requested, instead of only the explicit one.

**Comment by joeyparrish** [comment]
Apparently fixing that typo fixed the bug... not sure how, but it's working correctly now.

**Pull request merged**